python/tests/builder/test_kernel_builder.py

```diff
@@ -879,120 +879,134 @@ def test_recursive_calls():
 
 def test_from_state():
```

Comment:
I think there's a better Pytest way to do this. @khalatepradnya might be able to point you in the right direction on how to create a Pytest decorator that does a conditional check before enabling the test

Reply:
Updated [66eb983](https://github.com/NVIDIA/cuda-quantum/pull/1508/commits/66eb983fe19a0557074f94a5a7d9b35255d464cd)